### PR TITLE
DM-12085: Camera geometry incorrect and outdated in obs_test

### PR DESCRIPTION
**Pull request by r-owen**
Add a test for plate scale

### CODE REVIEW

**Comment by timj** [comment]
@r-owen if you can get this package flake8-clean you can enable flake8 testing by adding a `setup.cfg`. Something to consider for any repo being updated.

File: python/lsst/obs/base/camera_tests.py [hunk 1]
```diff
@@ -77,3 +84,30 @@ def test_camera_butler(self):
         self.assertEqual(camera.getName(), self.camera_data.camera_name)
         self.assertEqual(len(camera), self.camera_data.n_detectors)
         self.assertEqual(next(iter(camera)).getName(), self.camera_data.first_detector_name)
+
+    def test_plate_scale(self):
+        """Check the plate scale at center of focal plane
+
+        Check plate_scale using the FOCAL_PLANE to FIELD_ANGLE transform
+        from the camera
```
Comment (parejkoj) [inline python/lsst/obs/base/camera_tests.py]: period at the end.

File: python/lsst/obs/base/camera_tests.py [hunk 2]
```diff
@@ -77,3 +84,30 @@ def test_camera_butler(self):
...
+        """
+        plate_scale = self.camera_data.plate_scale
+        self.assertIsNotNone(plate_scale)
+        camera = self.butler.get('camera', immediate=True)
+        focalPlaneToFieldAngle = camera.getTransformMap().getTransform(FOCAL_PLANE, FIELD_ANGLE)
+        focalPlaneRadiusMm = 0.001
```
Comment (parejkoj) [inline python/lsst/obs/base/camera_tests.py]: Here's a definite case where I wish we had astropy.quantity compatibility. I'm assuming this is to convert mm to meters?

File: python/lsst/obs/base/camera_tests.py [hunk 3]
```diff
@@ -77,3 +84,30 @@ def test_camera_butler(self):
...
+        for offsetAngleRad in (0.0, 0.65, 1.3):
```
Comment (parejkoj) [inline python/lsst/obs/base/camera_tests.py]: Why these values in particular?

File: python/lsst/obs/base/camera_tests.py (hunk 2 above)
Comment (r-owen) [reply python/lsst/obs/base/camera_tests.py]: There is no conversion being done; focal plane coordinates are in mm (and field angle coordinates are in radians). I picked a radius small enough so as to be in the linear regime, even if the camera has significant optical distortion.

File: python/lsst/obs/base/camera_tests.py (hunk 3 above)
Comment (r-owen) [reply python/lsst/obs/base/camera_tests.py]: I just picked a few angles in a circle.

**Comment by r-owen** [comment]
@timj that sounds great, but unfortunately we can't enable flake8 until DM-12041 is fixed.

**Comment by timj** [comment]
you can enable flake8 but mask specific warnings per-file for testing only.

File: python/lsst/obs/base/camera_tests.py (hunk 2 above)
Comment (parejkoj) [reply python/lsst/obs/base/camera_tests.py]: Ah. Please make a comment to that effect. This looked like a conversion factor to me.

**Pull request merged**